changelogs/unreleased/gh-5295-split-brain-detection.md

```diff
@@ -0,0 +1,6 @@
+## feature/raft
+
```

Comment:
Checkpatch doesn't see the doc request for some reason. But I couldn't understand what's wrong exactly.
I remember there was an issue with checkpatch, when it stopped checking the commit message right on `---`.
Probably because `---` is a separator between the commit message and the diff itself in `git format-patch` output.

I don't have `---` in the message, so there's some other issue. But I didn't bother looking for the exact line which causes checkpatch to stop